### PR TITLE
vendor: bump google/certificate-transparency-go to 1.0.20

### PR DESCRIPTION
**Pull request by kolyshkin**
This is to include the Windows + Go1.11 fix (https://github.com/google/certificate-transparency-go/issues/284).

It was part of https://github.com/moby/moby/pull/37358

### CODE REVIEW

**Review by boaz0 — APPROVED** [review]
LGTM :+1: if all :green_heart:

**Comment by thaJeztah** [comment]
Equivalent PR's in SwarmKit; https://github.com/docker/swarmkit/pull/2688, and https://github.com/docker/swarmkit/pull/2689

I was planning to have it updated in SwarmKit first, then here together with a SwarmKit bump, but looks like it takes a bit to get it merged; I see enough LGTM's already so, I'm good with going ahead here

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**
